testing/jest/button.tests.tsx

```diff
@@ -19,7 +18,7 @@ import type ButtonRef from '../../js/renovation/button.p';
 import Icon from '../../js/renovation/icon.p';
 import Widget from '../../js/renovation/widget.p';
 import type { WidgetProps } from '../../js/renovation/widget';
-import type { ButtonProps } from '../../js/renovation/button';
+import DButton, { ButtonProps } from '../../js/renovation/button';
```

Comment:
I think we should not use `D` prefix. In the future, all our tests will be declaration tests. And the `D` prefix will be excess.

Reply:
OK, I see the `p` file here.